contrib/kind.yaml.j2

```diff
@@ -0,0 +1,76 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+networking:
+  # the default CNI will not be installed
+  disableDefaultCNI: true
+  apiServerAddress: {{ ovn_apiServerAddress | default('11.12.13.1') }}
+  apiServerPort: 11337
+{%- if ovn_ip_family %}
+  ipFamily: {{ ovn_ip_family }}
+{%- endif %}
+kubeadmConfigPatches:
+- |
+{%- if ovn_ip_family is equalto "DualStack" %}
+  apiVersion: kubeadm.k8s.io/v1beta2
+  featureGates:
+    IPv6DualStack: true
+{%- endif %}
+  kind: ClusterConfiguration
+  metadata:
+    name: config
+  apiServer:
+    extraArgs:
+{%- if ovn_ip_family is equalto "DualStack" %}
+      "feature-gates": "SCTPSupport=true,IPv6DualStack=true"
```

Comment:
we've simplified the feature-gate configuration in KIND now,now you just only need to configure the featureñgates in the kind config file and it cascades it to all the kubernetes components

https://github.com/kubernetes-sigs/kind/blob/1df6ecb9781505f755d9516077a4a9f3368f20bb/hack/ci/e2e-k8s.sh#L118-L137

Reply:
Done.
Ripped all the but the IP Family out. much cleaner. Thanks.